dev/integration_tests/deferred_components_test/run_release_test.sh

```diff
@@ -49,7 +49,7 @@ do
         echo "Failure: Deferred component did not load."
         exit 1
```

Comment:
btw, this won't exit. It would need to call `pkill -P $$` prior to `exit 1` in my local env.